psalm-baseline.xml

```diff
@@ -1,8 +1,90 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="5.24.0@462c80e31c34e58cc4f750c656be3927e80e550e">
+<files psalm-version="6.10.0@9c0add4eb88d4b169ac04acb7c679918cbb9c252">
+  <file src="src/AbstractAdapterOptionsTest.php">
+    <MissingOverrideAttribute>
+      <code><![CDATA[protected function setUp(): void]]></code>
+    </MissingOverrideAttribute>
+  </file>
+  <file src="src/AbstractCacheItemPoolIntegrationTest.php">
+    <MissingOverrideAttribute>
```

Comment:
Nitpick, can you please add:

```xml
<MissingOverrideAttribute>
    <errorLevel type="suppress">
       <directory name="src" />
       <directory name="test" />
    </errorLevel>
</MissingOverrideAttribute>
```

This prevents the baseline filling up with irrelevant issues and hidding the ones that matter.